### PR TITLE
Show only latest resource by default in discovery

### PR DESCRIPTION
**Pull request by amitaibu**
#214

Still need to add a test.

### CODE REVIEW

**Comment by amitaibu** [comment]
@mateu-aguilo-bosch @mirzu Here's indeed a place where a human-readable hint about the `?all=true` would be nice.

Question is where to add this? I would expect in the formatter's prepare function, but the Restful handler doesn't (yet?) have control over it.

**Comment by amitaibu** [comment]
@mateu-aguilo-bosch tests are green.

File: tests/RestfulDataProviderCToolsPluginsTestCase.test [hunk 1]
```diff
@@ -27,6 +27,20 @@ class RestfulDataProviderCToolsPluginsTestCase extends DrupalWebTestCase {
 
     // Assert sorting and filtering works as expected.
     $request = array(
+      // Get all resources.
+      'filter' => array(
+        'resource' => array(
+          'value' => 'articles',
+          'operator' => '=',
```
Comment (e0ipso) [inline tests/RestfulDataProviderCToolsPluginsTestCase.test:34]: Is this necessary? Anyways, I like being declarative.

**Comment by e0ipso** [comment]
Fantástico!

Merging.

**Pull request closed without merging**